SU-HSDO/suhumsci
# SHS-5841: The collection title on a wider screen is misaligned

# READY FOR REVIEW

## Summary
Fix the collection title alignment on a wider screen

## Need Review By (Date)
10/08

## Urgency
low

## Steps to Test
1. Add a Collection to a Flexible Page
2. Select the visible as a Heading 2 option under Title settings
3. Set Background Color (which is getting updated to Background Width) to Full Width
4. Verify the title is aligned to the rest of the content

<img width="1512" alt="Screenshot 2024-10-01 at 1 40 56 PM" src="https://github.com/user-attachments/assets/dd1fbc1a-73ee-42f5-98de-31125f426c19">


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208463191591251
